index.html

```diff
@@ -417,42 +384,57 @@
     };
 
     callback ThingEventListener = void (ConsumedThing thing);
+
+    enum ConsumedThingEvents {
+        "propertychanged",
+        "propertyremoved",
+        "propertyadded",
+        "actionchanged"
```

Comment:
I think it's simpler to write code for simple events. There is only 6 of them.

But I like the idea of having one event for `descriptionchange`. It is more expressive and creates less confusion (for instance with `propertychange` it is not clear whether the property value has changed, or the type, or the access). Of course we could have both `propertychange` (for TD) and `propertyupdate` (for value). But it would be more clear to have one `descriptionchange` event for TD changes for now.